src/savefilterdialog.cpp

```diff
@@ -49,7 +49,7 @@ bool SaveFilterDialog::SaveFilter(const QString& filename)
     qDebug() << "attempting to save...";
 
     // Validate filename
-    QRegularExpression regex("^[a-z0-9_]+$", QRegularExpression::CaseInsensitiveOption);
+    QRegularExpression regex("^[a-z0-9-_ ]+$", QRegularExpression::CaseInsensitiveOption);
```

Comment:
I think it would be safer if we escape the dash, dash can also be used to indicate ranges of characters, like in "a-z". 
Proposed regex:
```
"^[a-z0-9\-_ ]+$"
```